engine/src/main/java/org/terasology/rendering/opengl/FBOConfig.java

```diff
@@ -52,59 +47,49 @@
      *                   Type.NO_COLOR will result in -no- color buffer attached to the FBO
      *                   (WARNING: this could result in an FBO with Status.DISPOSED - see FBO.getStatus()).
      */
-    public FBOConfig(String title, FBO.Dimensions dimensions, FBO.Type type) {
-        this.title = title;
+    public FBOConfig(ResourceUrn resourceUrn, FBO.Dimensions dimensions, FBO.Type type) {
+        this.resourceUrn = resourceUrn;
         this.dimensions = dimensions;
         this.type = type;
-        this.isStatic = false;
-        isRelativeToScreenSize = false;
     }
 
     /**
      * Same as the previous FBObuilder constructor, but taking in input
      * explicit, integer width and height instead of a Dimensions object.
      */
-    public FBOConfig(String title, int width, int height, FBO.Type type) {
-        this(title, new FBO.Dimensions(width, height), type);
+    public FBOConfig(ResourceUrn resourceUrn, int width, int height, FBO.Type type) {
+        this(resourceUrn, new FBO.Dimensions(width, height), type);
     }
 
-    public FBOConfig(String title, float scale, FBO.Type type) {
+    public FBOConfig(ResourceUrn resourceUrn, float scale, FBO.Type type) {
         Preconditions.checkArgument(scale != 0, "scale can not be zero.");
-        this.title = title;
+        this.resourceUrn = resourceUrn;
         this.type = type;
         this.scale = scale;
-        this.isRelativeToScreenSize = true;
     }
 
-    public FBOConfig(String title, float scale, FBO.Type type, boolean isStatic) {
-        this(title, scale, type);
-        this.isStatic = isStatic;
+    public FBOConfig(ResourceUrn resourceUrn, FBO.Type type) {
+        this.resourceUrn = resourceUrn;
+        this.type = type;
     }
 
-    public boolean isFBODynamic() {
-        return !this.isStatic;
-    }
-/*
- *  * @param useDepthBuffer If true the FBO will have a 24 bit depth buffer attached to it. (GL_DEPTH_COMPONENT24, GL_UNSIGNED_INT, GL_NEAREST)
+   /*
+    * @param useDepthBuffer If true the FBO will have a 24 bit depth buffer attached to it. (GL_DEPTH_COMPONENT24, GL_UNSIGNED_INT, GL_NEAREST)
     * @param useNormalBuffer If true the FBO will have a 32 bit normals buffer attached to it. (GL_RGBA, GL_UNSIGNED_BYTE, GL_LINEAR)
     * @param useLightBuffer If true the FBO will have 32/64 bit light buffer attached to it, depending if Type is DEFAULT/HDR.
-*                       (GL_RGBA/GL_RGBA16F_ARB, GL_UNSIGNED_BYTE/GL_HALF_FLOAT_ARB, GL_LINEAR)
+    *                       (GL_RGBA/GL_RGBA16F_ARB, GL_UNSIGNED_BYTE/GL_HALF_FLOAT_ARB, GL_LINEAR)
     * @param useStencilBuffer If true the depth buffer will also have an 8 bit Stencil buffer associated with it.
     *                         (GL_DEPTH24_STENCIL8_EXT, GL_UNSIGNED_INT_24_8_EXT, GL_NEAREST)
-                *                         */
-
-    public FBOConfig setStatic() {
-        this.isStatic = true;
-        return this;
-    }
+    *
+    */
```

Comment:
Didn't we mention in a previous commit or PR that this disconnected block can be removed from here as it duplicates something written elsewhere, in the right place?